### DISCUSSION TITLE
Mixing sounds

### DISCUSSION THREAD

**Discussion by EldarFarkhutdinov** [schreibfaul1/ESP32-audioI2S#223] [Q&A]
Good afternoon dear!

I am making a game on esp32.
Tell me please, сan I play MP3 background music and wav audio at the same time?

**Comment by qniens** [comment]
I like to rephrase this question to make it more generic: Is polyphonic playback supported by this library?

**Comment by schreibfaul1** [comment]
It is not possible to open several audio sources at the same time.

**Reply by EldarFarkhutdinov** [reply]
Thank you very much for the answer!
